### ISSUE TITLE
Fix country data

### ISSUE THREAD

**Issue by dsbb-finddx** [dsbbfinddx/FINDCov19TrackerData#167]
- [ ] El Salvador issue with scraping and negative value on 2021-09-13
- [ ] Estonia issue with negative value on 2021-07-28. Scraping works but corrected columns keep the same since 2021-07-28

**Comment by dsbb-finddx** [comment]
- [ ] United States: still not scraping data correctly. Please use below code and update accordingly cumulative test numbers:
``` {r}
us_test_data <- readr::read_csv("https://raw.githubusercontent.com/govex/COVID-19/master/data_tables/testing_data/time_series_covid19_US.csv",
                                    col_types = readr::cols(), quoted_na = FALSE
)
us_test_data$date <- as.Date(us_test_data$date, format = '%m/%d/%Y')

us_test_data_per_day <- us_test_data %>%
  dplyr::group_by(date) %>%
  dplyr::arrange(date) %>%
  dplyr::summarise(cumulative_all_tests = sum(tests_combined_total))

```


- [ ] UK data needs to be updated using the API instead of scraping. Check how to [here](https://coronavirus.data.gov.uk/details/download ) and [here](https://coronavirus.data.gov.uk/details/developers-guide/main-api).
Permalink to use for cumulative number of tests by date (change with current date at url): https://api.coronavirus.data.gov.uk/v2/data?areaType=overview&metric=cumTestsByPublishDate&format=csv&release=2022-01-24

- [ ]  LaoPeoplesDemocraticRepublic: 2021-07-12 neg value
- [ ] Latvia: 2021-09-07 neg value, scraping works erroneously. Change source to [this](https://data.gov.lv/dati/eng/dataset/covid-19/resource/d499d2f0-b1ea-4ba2-9600-2c701b03bd4a) and recreate time series as per data column: Number of Tests - Indicates the number of tests performed in the last 24 hours.

- [ ] Israel data: negative value on 2021-07-01, erroneous scraping. Recreate time series using downloaded excel from [dashboard](https://datadashboard.health.gov.il/COVID-19/general), using dates from the beginning.
![Screenshot 2022-01-19 at 00 40 44](https://user-images.githubusercontent.com/62348391/150036420-d6e54fc9-4722-4c4d-adb5-686e004418e1.png)

- [ ] Grenada: negative value on 2021-12.29. Update manually cumulative value of tests using [source](https://www.facebook.com/photo?fbid=235543305428063&set=a.189134130068981)

- [ ] Andorra: scraping works but not updated corrected values since 2021-10-14. Another negative value on 2021-11-26

**Comment by angelicambg** [comment]
@dsbb-finddx Israel Fixed.

Benin in Africa CDC dashboard has a lower value than the value reported last time in https://www.gouv.bj/coronavirus/ and at some point, they didn't report anymore. Should I change the previous values to have the same value as Africa CDC dashboard?

**Comment by dsbb-finddx** [comment]
> @dsbb-finddx Israel Fixed.
> 
> Benin in Africa CDC dashboard has a lower value than the value reported last time in https://www.gouv.bj/coronavirus/ and at some point, they didn't report anymore. Should I change the previous values to have the same value as Africa CDC dashboard?

Yes, since Africa CDC keeps on reporting on Benin, change the source and values as per new source.

**Comment by dsbb-finddx** [comment]
- [ ] Ireland needs to be updated. Is scraping working correctly, because the [source](https://covid-19.geohive.ie/pages/hospitals-icu--testing) gets updated frequently, but we have not updated since 30/12/2021. Use [csv](https://covid-19.geohive.ie/datasets/f6d6332820ca466999dbd852f6ad4d5a_0/explore?location=53.438887%2C-8.057012%2C7.72&showTable=true) file to update all time series from 30/12/2021 to today, but try to scrape if possible.

**Comment by dsbb-finddx** [comment]
> * [ ]  Ireland needs to be updated. Is scraping working correctly, because the [source](https://covid-19.geohive.ie/pages/hospitals-icu--testing) gets updated frequently, but we have not updated since 30/12/2021. Use [csv](https://covid-19.geohive.ie/datasets/f6d6332820ca466999dbd852f6ad4d5a_0/explore?location=53.438887%2C-8.057012%2C7.72&showTable=true) file to update all time series from 30/12/2021 to today, but try to scrape if possible.

@angelicambg @seawaR could you please let me know when will the historical data be updated?
